articles/express/index.md

```diff
@@ -0,0 +1,318 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/express/
+title: Introduction to Express.js
+description: Express is a lightweight framework on top of Node.js, it adds functionality like middleware, template engines, and routing.
+author: rohan-reddy
+date: 2020-09-18T00:00:00-11:00
+topics: [Node.js]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/express/hero.jpg
+    alt: expressjs example text image
+---
+[Node.js](http://nodejs.org/) is a JavaScript run time environment which is used to create server-side applications and tools. Node.js is fast, portable and written in JavaScript. Node.js does not directly support common tasks such as handling requests, serving files, handling HTTP verbs (`GET`, `POST`, …), this is where Node.js rich ecosystem comes to our aid.
+<!--more-->
+
+**Express.js (Express) is a light web framework which sits on top of Node.js and it adds functionality ([middleware](https://expressjs.com/en/guide/writing-middleware.html) , [routing](https://en.wikipedia.org/wiki/Routing), etc.) and simplicity to Node.js.**
+
+When creating a Node.js web application, we write a single JavaScript application which listens to requests from the browser, based on the request the function will send back some data or an HTML web page.
+
+![flow of a request](/engineering-education/express/node.png)
+
+[Image Credit](https://www.manning.com/books/express-in-action)
+
+A *request handler* is a JavaScript function which takes a request and sends an appropriate response.
+
+Node.js APIs can get complex and writing how to handle a single request can end up being to 50 lines of code. Express makes it easier to write web applications with Node.js.
```

Comment:
being to doesn't make sense. Should it be over? Consider rephrasing to easier to write **Node.js web applications** to improve flow.